indy_common/test/types/test_nym.py

```diff
@@ -0,0 +1,32 @@
+from plenum.common.constants import TARGET_NYM, TXN_TYPE
+import pytest
+from indy_common.constants import DIDDOC_CONTENT, NYM
+from indy_common.types import ClientNYMOperation
+
+VALID_TARGET_NYM = 'a' * 43
+
+
+@pytest.fixture
+def validator():
+    yield ClientNYMOperation()
+
+
+def test_nym(validator):
+    msg = {
+        TXN_TYPE: NYM,
+        TARGET_NYM: VALID_TARGET_NYM,
+        DIDDOC_CONTENT: '{}',
+    }
+    validator.validate(msg)
+
+
+def test_nym_raw_dictionary(validator):
+    msg = {
+        TXN_TYPE: NYM,
+        TARGET_NYM: VALID_TARGET_NYM,
+        DIDDOC_CONTENT: {},
+    }
+    with pytest.raises(TypeError) as ex_info:
+        validator.validate(msg)
+
+    ex_info.match("validation error")
```

Comment:
I'm not clear on how these tests are different.  I'm sure I'm missing something.  Could you elaborate?

Reply:
`'{}'` versus `{}`, simply ensuring that the nym transaction accepts only JSON strings as DID Doc content and not dictionaries. This was actually more of a regression test than anything else; earlier iterations of the did:indy changes were expecting dictionaries over JSON strings.

Reply:
Now that you've pointed that out, it obvious.  I completely missed that at first.  I'm wondering if a comment to that affect on the tests would help others as well.

Reply:
Docstrings added for the two test methods!